python/understack-workflows/understack_workflows/helpers.py

```diff
@@ -35,6 +36,46 @@ def arg_parser(name):
     return parser
 
 
+def undersync_switch_parser(name):
```

Comment:
Should this be in the generic helpers? This is the CLI parser for one command. These helpers are shared by all the commands?

Reply:
moved to a main file in f6b295bbe5a1